libcloud/compute/drivers/linode.py

```diff
@@ -728,6 +752,912 @@ def _to_nodes(self, objs):
         return list(nodes.values())
 
 
+class LinodeNodeDriverV4(LinodeNodeDriver):
+
+    connectionCls = LinodeConnectionV4
+    _linode_disk_filesystems = LINODE_DISK_FILESYSTEMS_V4
+
+    LINODE_STATES = {
+        'running': NodeState.RUNNING,
+        'stopped': NodeState.STOPPED,
+        'provisioning': NodeState.STARTING,
+        'offline': NodeState.STOPPED,
+        'booting': NodeState.STARTING,
+        'rebooting': NodeState.REBOOTING,
+        'shutting_down': NodeState.STOPPING,
+        'deleting': NodeState.PENDING,
+        'migrating': NodeState.MIGRATING,
+        'rebuilding': NodeState.UPDATING,
+        'cloning': NodeState.MIGRATING,
+        'restoring': NodeState.PENDING,
```

Comment:
There is a missing state 'resizing'. I checked the API documentation and it does not appear. But I made some tests and it does exist. May be a good map value could be:
'resizing': NodeState.RECONFIGURING